config/reader.go

```diff
@@ -39,13 +39,15 @@ func getAgentInfo() (*apppb.AgentInfo, error) {
 	if err != nil {
 		return nil, err
 	}
+	platform := fmt.Sprintf("%s/%s", runtime.GOOS, runtime.GOARCH)
 
 	return &apppb.AgentInfo{
 		Host:        hostname,
 		Ips:         ips,
 		Os:          runtime.GOOS,
 		Version:     Version,
 		GitRevision: GitRevision,
+		Platform:    &platform,
```

Comment:
The rdk does not have tests for `AgentInfo`.

Happy to add some if others think it is worthwhile.

Reply:
https://viam.atlassian.net/browse/APP-2276